src/systems/render-manager/hubs-batch-raw-uniform-group.js

```diff
@@ -71,7 +65,7 @@ export default class HubsBatchRawUniformGroup extends BatchRawUniformGroup {
         instanceId,
         mesh.visible &&
         (mesh.parent && mesh.parent.visible) &&
-        (!cameraSystem.inspected || inParentHierarchyOf(cameraSystem.inspected, mesh))
+        (!inspected || inspectedMeshesFromBatch.indexOf(mesh) !== -1)
```

Comment:
depending on how hot the loop is here, it may be smarter to put a top level branch like:
```
if (!inspected) {
  // Common code path, with mesh.matrixWorld and no conditional on inspectedMeshesFromBatch
} else {
  // the code you have now (minus any redundant checking on inspected
}
```
since most of the time the first condition will be hit, skipping the array walks through inspectedMeshesFromBatch etc